### ISSUE TITLE
Keyboard control over Bootstrap Dropdown components stops working

### ISSUE THREAD

**Issue by daniel-mf** [snapappointments/bootstrap-select#2308]
When I add this plugin, keyboard control of bootstrap dropdown components stops working.
If I remove `bootstrap-select.js`, they start working again.

* Bootstrap v4.1.3
* Bootstrap-select v1.13.10

You can see the example in [this test case](https://plnkr.co/edit/gMLPnkMsHW0fX4LuVuyy?p=preview).

**Comment by daniel-mf** [comment]
I've figured out that this plugin intentionally disables keyboard control of bootstrap dropdown by calling `$(document).off('keydown.bs.dropdown.data-api')`

**Comment by daniel-mf** [comment]
I fixed this by adding:
```javascript
$(document).on(
    'keydown.bs.dropdown.data-api',
    '.dropdown-menu[role="menu"], [data-toggle="dropdown"]:not([role="combobox"])',
    bootstrap.Dropdown._dataApiKeydownHandler
);
```

Just gotta remember to add `role="menu"` to my Dropdowns

**Comment by caseyjhol** [comment]
Released in [v1.13.12](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.12)!

**Comment by Realex-fire** [comment]
@caseyjhol 
Your solution is not entirely correct. keydown.bs.dropdown.data-api will still be attaching to .bootstrap-select .dropdown-menu since the original event is attaching to .dropdown-menu.
You can see this by looking at _$._data(document, 'events')_
In order for everything to work correctly, the event needs to be re-attaching, as suggested by @daniel-mf

**Comment by Realex-fire** [comment]
@caseyjhol
Your solution is not entirely correct. `keydown.bs.dropdown.data-api` will still be attaching to `.bootstrap-select .dropdown-menu` since the original event is attaching to `.dropdown-menu`.
You can see this by looking at `$.data(document, 'events')`.
In order for everything to work correctly, the event needs to be re-attaching, as suggested by @daniel-mf

**Comment by Realex-fire** [comment]
@caseyjhol
Your solution is not entirely correct. `keydown.bs.dropdown.data-api` will still be attaching to `.bootstrap-select .dropdown-menu` since the original event is attaching to `.dropdown-menu`.
You can see this by looking at `$.data(document, 'events')`.
In order for everything to work correctly, the event needs to be re-attaching, as suggested by @daniel-mf

**Comment by Realex-fire** [comment]
@caseyjhol
Your solution is not entirely correct. `keydown.bs.dropdown.data-api` will still be attaching to `.bootstrap-select .dropdown-menu` since the original event is attaching to `.dropdown-menu`.
You can see this by looking at `$.data(document, 'events')`.
In order for everything to work correctly, the event needs to be re-attach, as suggested by @daniel-mf

**Comment by Realex-fire** [comment]
@caseyjhol
Your solution is not entirely correct. `keydown.bs.dropdown.data-api` will still be attaching to `.bootstrap-select .dropdown-menu` since the original event is attaching to `.dropdown-menu`.
You can see this by looking at `$._data(document, 'events')`.
In order for everything to work correctly, the event needs to be re-attach, as suggested by @daniel-mf

**Comment by JPBetley** [comment]
Seconded what @Realex-fire says here. Observed behavior of using v1.13.12 in our project has shown that the "active" option only changes on first keydown, does not update on subsequent keydown events

**Comment by caseyjhol** [comment]
Released in [v1.13.13](https://github.com/snapappointments/bootstrap-select/releases/tag/v1.13.13)!

**Comment by grreeenn** [comment]
still facing this in 1.13.18 (BS 3.4.1, jQuery 3.7.1)
